### ISSUE TITLE
Creating the first account don't work and Viziquer cannot be used locally 

### ISSUE THREAD

**Issue by vemonet** [LUMII-Syslab/viziquer#8]
Hi, I have tried to start Viziquer locally

I used the `docker-compose-public.yml` file as instructed in the README, I run Ubuntu 20.04, with Docker version 20.10.8

I entered my email when signing up as the first user, it is now asking for email confirmation, but it did not send an email tot he address I entered. It was quite surprising to meet this step, since there is no public instance of this service, and even the demos I saw were using local deployment of Viziquer (and I have never seen a service deployed locally that also requires to confirm email)

Unfortunately I did not received any email (I also checked my junks folder, but I guess I need to configure a local email server so that the local Viziquer can send an email for the confirmation.)

How can this email verification be disabled or fixed? Note that ultimately the goal is to be able to deploy and use Viziquer

**Comment by karlisc** [comment]
Sorry for the confusion, we need to fix it. Meanwhile, your account has been created and you can log into it.

**Comment by vemonet** [comment]
Thanks, I can access it now! But that means that even local instance require to connect to a central server?

**Comment by karlisc** [comment]
No, it does not connect. The form for account creation that involves messages about e-mail checking has been used from the platform-level templates as we thought initially that this might be a good idea to have. The actual e-mail sending and checking have never been implemented, so confusion is left. We have to fix this.

**Comment by vemonet** [comment]
Ok, thanks for the details! Should I let this issue open until you fixed it?

**Comment by karlisc** [comment]
Sure, this will help us give it an appropriate priority.

**Comment by karlisc** [comment]
Fixed. E-mail checking not performed and not initiated.
